Assignment3.txt

```diff
@@ -0,0 +1,10 @@
+Determining Complexity
+1.
+	a. n
+	b. n^2+n
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms, so this just becomes O(n^2).